scripts/pylib/twister/twisterlib/sca.py

```diff
@@ -0,0 +1,34 @@
+# vim: set syntax=python ts=4 :
```

Comment:
I don't think we generally add editor settings in files, although I give a :+1: for the choice of editor.

```suggestion
```

Reply:
+1 - I think the accepted practice is just to run files through ``black file.py -l 100``